packages/w3up-client/README.md

```diff
@@ -156,7 +156,7 @@ await client.setCurrentSpace(space.did())
 
 #### Delegating from Space to Agent
 
-In order to store data with w3up, your Agent will need a delegation from a Space. This automatically happens if you called `login(email)` then `registerSpace()`. However, if you are initializing the client with a previously created Space, you can `login(email)` then claim a delegation granted to the account associated with your email:
+In order to store data with w3up, your Agent will need a delegation from a Space. This automatically happens if you called `Account#provision` on a space. However, if you are initializing the client with a previously created Space, you can `login(email)` then claim a delegation granted to the account associated with your email:
```

Comment:
I think login does this automatically - i.e. no need to call `await capability.access.claim()`

Reply:
https://github.com/web3-storage/w3up/pull/1218/commits/6c4262793307ea72420ea623a4da16cb67eff26d
https://github.com/web3-storage/w3up/pull/1218/commits/3fd0800fa6c9f1db1bbc895daa0bd2432f2357e7